### PR TITLE
fix(region): optimized host-list speed

### PR DESCRIPTION
**Pull request by ioito**
**What this PR does / why we need it**:

- [ ] Smoke testing completed

If yes, write the release branches name in the below format and submit the related cherry-pick PR:
- release/3.11

### CODE REVIEW

File: pkg/compute/models/hosts.go [hunk 1]
```diff
@@ -4553,29 +4831,28 @@ func (hh *SHost) isRedfishCapable() bool {
 }
 
 func (hh *SHost) canPrepare() error {
-	if !hh.IsBaremetal {
```
Comment (zexi) [inline pkg/compute/models/hosts.go:4556]: 这个 canPrepare 函数还会被 getMoreDetails 函数调用，把里面的条件去掉会不会有问题？
image

Comment (ioito) [reply pkg/compute/models/hosts.go:4556]: 我已经删掉getMoreDetails函数了，这个改动主要就是去掉 getMoreDetails 避免每个host的资源都sql查询数据

**Pull request merged**